prober/http_test.go

```diff
@@ -257,12 +353,15 @@ func TestFailIfBodyMatchesRegexp(t *testing.T) {
 		t.Fatal(err)
 	}
 	expectedResults := map[string]float64{
-		"probe_failed_due_to_regex": 1,
+		"probe_failed_due_to_regex":           1,
+		"probe_http_content_length":           float64(len(msg)), // issue #673, check that this is correctly populated with regex validations
```

Comment:
All these comments need formatting too.

Reply:
Thanks for spotting this.

While fixing this I noticed that the other two cases should be checking the results, too. I added that, and since the code was getting a little repetitive, I reorganized the code in the test a little. I simply moved the first block of code into an anonymous function and added t.Run to call that for each test case.

Solved.